README.md

```diff
@@ -1,7 +1,390 @@
-# sekai.team
+![tailwind-nextjs-banner](/public/static/images/twitter-card.png)
```

Comment:
Should we keep the original README.md content from `tailwind-nextjs` as a secondary content?

Reply:
I'll make a custom README which honors some of timlrx's original README content